core/api/serializers.py

```
@@ -1,12 +1,208 @@
+    Serializer for Contribution model.
+    '''
+    videos = ContributionVideoSerializer(many=True, required=False)
+    tags = ContributionTagSerializer(many=True, required=False)
+    origine = ContributionOriginSerializer(many=True, required=False)
+    notes = ContributionNoteSerializer(many=True, required=False)
+
+    class Meta:
+        model = Contributions
+        fields = '__all__'
+
+    def create(self, validated_data):
```

Ruff (0.8.2)

`create` is too complex (13 > 10)

(C901)